File: languages/php/Bloock/Entity/Identity/Attribute.php
<?php

namespace Bloock\Entity\Identity;

/**
 * Represents an attribute with an identifier and a corresponding value.
 */
class Attribute
{
    public string $id;
    public $value;

    /**
     * Constructs an Attribute object with the specified parameters.
     * @param string $id
     * @param $value
     */
    public function __construct(string $id, $value)
    {
        $this->id = $id;
        $this->value = $value;
    }
}

File: languages/java/src/main/java/com/bloock/sdk/entity/identity/Attribute.java
package com.bloock.sdk.entity.identity;

/**
 * Represents an attribute with an identifier and a corresponding value.
 * @param <T>
 */
public class Attribute<T> {
  String id;
  T value;

  /**
   * Constructs an Attribute object with the specified parameters.
   * @param id
   * @param value
   */
  public Attribute(String id, T value) {
    this.id = id;
    this.value = value;
  }
}
